boot/bootdata/autorun.inf

```diff
@@ -1,3 +1,3 @@
 [autorun]
-shellexecute=readme.txt
+open=setup.exe
```

Comment:
For the reference: this part was affected by recent 03d4100.

Reply:
Exactly, and since this PR is not meant to have the GUI setup exposed that much when booting the bootcd as a livecd (or getting it autorun'd), I would like to have your point of view @oleg-dubinskiy and @whindsaks :
Do you think it would be better to keep the readme.txt starting, instead of the welcome.exe (renamed as setup.exe here) ?

Reply:
I personally would prefer to mimic Windows XP/2003 behaviour here. As I remember, when mounting the XP/2003 ISOs in Windows, the some kind of welcome window (with setup and other useful post-install tasks) is opening by default there, so I think we should do the same.
Also it looks more reasonable from the logical point of view, to prompt the user to setup, instead of just showing the general information, because if the user has mounted the ISO, most likely he wants to install the OS from it, not just read something about it.
So my opinion is to keep it as-is.

Reply:
Yes that's what I would do (and what this autorun does), _**when**_ a working GUI setup is included.

Reply:
The long term solution should be welcome.exe yes but for now, it is pretty useless because

- Autorun in ROS does not work (the driver/storage stack does not inform the shell)
- It does not support upgrading the existing ROS install.

Reply:
> It does not support upgrading the existing ROS install.

It's mostly this point that makes me lean towards @whindsaks's argument (and me reverting the autorun.inf change of this PR back to what's currently in master)